src/Config/RabbitMQOptions.cs

```diff
@@ -18,9 +18,5 @@ public class RabbitMQOptions
         public string ConnectionString { get; set; }
 
         public int Port { get; set; }
-
-        public string Exchange { get; set; }
-
-        public IBasicProperties Properties { get; set; }

```

Comment:
We should let users configure IBasicProperties . Please open an issue to address this in a separate PR.